.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  pull_request:
+    branches: ['*']
+  push:
+    branches: [develop, main]
+
+jobs:
+  audits:
+    name: Security Audits
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install Ruby and gems
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - name: Security audit dependencies
+        run: bundle exec bundler-audit --update
+      - name: Security audit ruby
+        run: bundle exec ruby-audit update && bundle exec ruby-audit check
```

Comment:
How is this different from `bundler-audit check` and is it worth adding a development dependency?

Reply:
`bundler-audit` checks the gems, in the gemfile.lock, while `ruby-audit` checks ruby.

`ruby_audit` complements `bundler-audit`, providing complete coverage for our Ruby stack